### PR TITLE
Milesone6:

### PR DESCRIPTION
**Pull request by Ambrosegithub**
## modal pop up window

- add js file.
- display the modal window using javascript.
- add images. 
- receive contribution from partners

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,173 +64,168 @@ <h1>AMBROSE ISIGBA</h1>
...
-        </div>
-        <div class="full-section-div fix">
-          <div id="img-div2" class="imj-project-section">
-            <img id="full-section-div2" src="./images/desktop2.png" alt="" />
-            <img id="list-of-project2-img" src="./images/desktop5.png" alt="" />
-            <img
-              id="list-of-project2-imgb"
-              src="./images/Group-83-copy.png"
-              alt=""
+ <section id="project-sections">
```
Comment (muneebulrehman) [inline index.html]: The whole projects section should be generated dynamically using Javascript. Kindly update this section accordingly.

Comment (muneebulrehman) [inline index.html]: Also, this whole section is a mess in desktop view. Have a look at the below screenshots. Kindly fix it and check it thoroughly before asking for another review.
image
image

File: index.html [hunk 2]
```diff
@@ -334,28 +330,70 @@ <h2>Get Started</h2>
...
+                  <button type="button" id="modal-btn1">see live <img id="cross-img4" src="./images/Union8.png" alt=""></button>
+                  <button type="button" id="modal-btn2">see source<img id ="cross-img3" src="./images/Vector1.png" alt=""></button>
+                </div>
+                <div class="modal-link">
+                  <a href=""> &#8592;  <span>previous project</span></a>
+                  <a href="">next project <span>&#8594;</span> </a>
+                </div>
+              </div>
+            </div>
+    <!--modal pop up ends here-->
```
Comment (muneebulrehman) [inline index.html]: Your model should also be generated dynamically using Javascript. Kindly update this section also.
Also, your model is spanning more than two pages vertically, and margins, padding, spacing, and positioning need to be almost exactly like the Figma template of your choosing. Kindly make it according to your Figma design.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Ambrosegithub , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @Ambrosegithub ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**